### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by cpanato**
- group dependabot updates
- update golangci-lint
- update go version in go.mod
- update check for public-services-are-unauthenticated resource

### CODE REVIEW

File: modules/regional-go-service/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+output "cloud_run_endpoint" {
+  description = "Cloud Run services endpoint"
+  value       = zipmap(values(google_cloud_run_v2_service.this).*.location, values(google_cloud_run_v2_service.this).*.uri)
+}
```
Comment (mattmoor) [inline modules/regional-go-service/outputs.tf:4]: While it's annoying, I had specifically avoided exposing this to ~force folks to go through `authorize-private-service`, and I'd kind of like to keep that.

However, you should be able to do something similar to that to look it up once this has deployed:
https://github.com/chainguard-dev/terraform-infra-common/blob/7112a5ef4814be6321fed4ef75f85ba54645bd37/modules/authorize-private-service/main.tf#L10-L16

**Review by mattmoor — COMMENTED** [review]
Really just one comment, but otherwise LGTM

File: modules/regional-go-service/outputs.tf (hunk 1 above)
Comment (cpanato) [reply modules/regional-go-service/outputs.tf:4]: got it, i will drop that

Comment (cpanato) [reply modules/regional-go-service/outputs.tf]: @mattmoor ptal

**Review by mattmoor — APPROVED** [review]

**Pull request merged**